Core/GDCore/Project/Project.cpp

```diff
@@ -865,6 +865,10 @@ void Project::ExposeResources(gd::ArbitraryResourceWorker& worker) {
   for (std::size_t j = 0; j < GetObjectsCount(); ++j) {
     GetObject(j).ExposeResources(worker);
   }
+
+  // Add loading screen background image if present
+  if (loadingScreen.GetBackgroundImageResourceName() != "")
+    worker.ExposeImage(loadingScreen.GetBackgroundImageResourceName());

```

Comment:
Excellent! 
It's a bit unusual to pass a string as reference, but it's how it works here and it's correct.